lib/mayaUsd/fileio/chaser/CMakeLists.txt

```diff
@@ -1,25 +1,29 @@
 # -----------------------------------------------------------------------------
 # sources
 # -----------------------------------------------------------------------------
-target_sources(${PROJECT_NAME} 
+target_sources(${PROJECT_NAME}
     PRIVATE
         chaser.cpp
         chaserRegistry.cpp
+        importChaserRegistry.cpp
```

Comment:
In translation framework code, we rarely use import or export in the names. Let's be consistent and use writer / reader convention.

Reply:
I'm less bothered by using "import" and "export" here since they're somewhat more tied to the import and export commands than to the translation framework, but whichever we choose, I think we should rename the existing `chaser.cpp` and `chaserRegistry.cpp` to include the "writer" or "export" terminology.

Reply:
I can certainly change the names as requested, but my concern is that the precedent is set in that the export chasers are already called that; it might be confusing to call these "reader" chasers instead afterwards (especially since the `readJob` calls methods such as `DoImport` as well, anyway, and the import chaser code runs at the end of that) that imply they do something other than symmetric functionality for the existing export chasers.

Reply:
@ysiewappl just to make sure I understood your proposal - you suggest we keep the current naming convention in the PR or you suggest to use ‘import’ and ‘export’ terminology in names?

Reply:
@kxl-adsk Yes, I'm suggesting we continue to use the terms "import" and "export" chasers and match that with the current naming convention in the PR (IMHO it's less confusing). But of course I can certainly rename these to be "reader" chasers as well; just as long as we decide one way or another and be consistent I have no big qualms with either.

Reply:
I see we all agree with the need for consistency. I won't generate more work than required, since both terminologies are already in use, i.e. I will give up on the switch to `reader`/`writer` but please rename `chaser.cpp` and `chaseRegistry.cpp` to include `export`.

Reply:
This is now addressed in ef5450c specifically as a separate commit, which should hopefully make it easier to revert if it causes a regression.